models/application.js

```diff
@@ -0,0 +1,224 @@
+const mongoose = require("mongoose");
+
+const { Schema } = mongoose;
+
+/**
+ * Dog Profile Model
+ */
+module.exports = mongoose.model(
+  "Application",
+  new Schema(
+    {
+      user: {
+        type: Schema.Types.ObjectId,
+        required: true,
+      },
+      firstName: {
+        type: String,
+        required: true,
+      },
+      lastName: {
+        type: String,
+        required: true,
+      },
+      address: {
+        addressOne: {
+          type: String,
+          required: true,
+        },
+        addressTwo: {
+          type: String,
+          required: true, // TODO should this be required?
+        },
+        city: {
+          type: String,
+          required: true,
+        },
+        state: {
+          type: String,
+          required: true,
+        },
+        country: {
+          type: String,
+          required: true,
+        },
+        zipcode: {
+          type: Number,
+          required: true,
+        },
+      },
+      email: {
+        type: String,
+        required: true,
+      },
+      dateOfBirth: {
+        type: String,
+        required: true,
+      },
+      homeType: {
+        type: String,
+        required: true,
+      },
+      landlord: {
+        // TODO should this also be required?
+        firstName: {
+          type: String,
+          required: true,
+        },
+        lastName: {
+          type: String,
+          required: true,
+        },
+        phone: {
+          type: String,
+          required: true,
+        },
+        email: {
+          type: String,
+          required: true,
+        },
+      },
+      fosterInfo: {
+        restrictions: {
+          type: String,
+          required: true,
+        },
+        sleepLocation: {
+          type: String,
+          required: true,
+        },
+        hoursAtHome: {
+          type: String,
+          required: true,
+        },
+        typicalDay: {
+          type: String,
+          required: true,
+        },
+        primaryCaregiver: {
+          type: String,
+          required: true,
+        },
+        othersOnboard: {
+          type: String,
+          required: true,
+        },
+        permissionToVisit: {
+          type: Boolean,
+          required: true,
+        },
+        pastExperience: {
+          type: String,
+          required: true,
+        },
+        whyFoster: {
+          type: String,
+          required: true,
+        },
+        oneMonthCommitment: {
+          type: String,
+          required: true,
+        },
```

Comment:
Should we make this a Boolean since the only options are "Yes" and "No"?

<img width="734" alt="image" src="https://user-images.githubusercontent.com/24444266/162107456-4fa6bb62-b79b-478a-a590-660eb0ada1f6.png">

Reply:
@dnanjunda thoughts?

Reply:
I believe there's supposed to be an "Other" option for this question. @petabite Maybe you missed this in your application page PR?

<img width="611" alt="Screen Shot 2022-05-12 at 10 08 50 PM" src="https://user-images.githubusercontent.com/57281768/168215167-749d71b6-e652-4cd4-80eb-df8f5b17f1fa.png">

Reply:
Oohh. Yeah, I must've missed this. I can add the 'Other' field in my PR.